lms/static/sass/multicourse/_sysadmin.scss

```diff
@@ -0,0 +1,47 @@
+.mgmt-commands-body #select-command{
+    border: solid 4px black;
+    padding: 20px;
+
+}
+
+.mgmt-commands-body .full-width{
+    width: 100%;
+}
+
+.mgmt-commands-body .bordered-fieldset{
+    border: solid 4px black;
+    padding: 20px;
+    margin-top: 30px;
+}
+
+.mgmt-commands-body .execute-command-button{
+    margin-top: 20px;
+}
+
+.mgmt-commands-body #command-description{
+    margin-top: 10px;
+}
+.mgmt-commands-body select{
```

Comment:
Does the select have a class? if yes, refer to it with that. If not, add a class and refer to it with that... ;)